tests/samplers_tests/test_partial_fixed.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+
+import pytest
+
+import optuna
+from optuna.samplers import PartialFixedSampler
+from optuna.samplers import RandomSampler
+from optuna.trial import Trial
+
+
+def test_params_identity() -> None:
+    def objective(trial: Trial) -> float:
+        x = trial.suggest_uniform("x", -10, 10)
+        y = trial.suggest_uniform("y", -10, 10)
+        return x ** 2 + y ** 2
+
+    study_0 = optuna.create_study()
+    study_0.sampler = RandomSampler(seed=42)
+    study_0.optimize(objective, n_trials=1)
+    x_sampled_0 = study_0.trials[0].params["x"]
+
+    # Fix parameter y as 0.0.
+    study_1 = optuna.create_study()
+    study_1.sampler = PartialFixedSampler(
+        fixed_params={"y": 0.0}, base_sampler=RandomSampler(seed=42)
+    )
+    study_1.optimize(objective, n_trials=1)
+    x_sampled_1 = study_1.trials[0].params["x"]
+    y_sampled_1 = study_1.trials[0].params["y"]
+    assert x_sampled_1 == x_sampled_0
+    assert y_sampled_1 == 0
+
+    # Fix parameter y as the -5.
+    study_2 = optuna.create_study()
+    study_2.sampler = PartialFixedSampler(
+        fixed_params={"y": -5}, base_sampler=RandomSampler(seed=42)
+    )
+    study_2.optimize(objective, n_trials=1)
+    x_sampled_2 = study_2.trials[0].params["x"]
+    y_sampled_2 = study_2.trials[0].params["y"]
+    assert x_sampled_2 == x_sampled_0
+    assert y_sampled_2 == -5
+
+
+@pytest.mark.parametrize("fixed_y", [0.5, 5, 5.5])
```

Comment:
I'm a bit confused with this test case. Is it necessary to test both `0.5` and `5.5`? Could you explain it?

Reply:
Thanks at all and sorry for the late reply @nzw0301 
You are right, `5.5` seems to be redundant. I'll delete it.